src/Dfc.CourseDirectory.Web/Views/BulkUploadApprenticeships/Index.cshtml

```diff
@@ -69,7 +69,12 @@
 
         <form method="post" id="bulkUploadForm" enctype="multipart/form-data" asp-controller="BulkUploadApprenticeships" asp-action="Index">
 
-            <p class="govuk-body">You can upload a CSV file to update your apprenticeship training courses, or you can @Html.ActionLink("add", "Index", "Apprenticeships"), @Html.ActionLink("edit or delete", "Index", "ProviderApprenticeships") each course individually.</p>
+            <p class="govuk-body">You can upload a CSV file to update your apprenticeship training courses, or you can:</p>
+
+            <ul class="govuk-list govuk-list--bullet">
+                <li>@Html.ActionLink("add a new", "Index", "Apprenticeships") apprenticeship training course</li>
```

Comment:
That link text doesn't make sense out of context, which is how a screen reader user would hear it. I understand screen reader users tab through all the links, and the reader reads out all the link texts, without any of the surrounding text.